paymentsheet/src/main/java/com/stripe/android/paymentsheet/ExternalPaymentMethodActivity.kt

```diff
@@ -0,0 +1,65 @@
+package com.stripe.android.paymentsheet
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.stripe.android.model.PaymentMethod
+
+/**
+ * Handles external payment methods.
+ *
+ * This will call the merchant's [ExternalPaymentMethodConfirmHandler.confirmExternalPaymentMethod] when initially
+ * created. When the merchant's implementation calls [ExternalPaymentMethodResultHandler.onExternalPaymentMethodResult],
+ * it will re-start this activity with the result. This class then finishes and the class that launched this activity
+ * is responsible for responding to that result.
+ */
+internal class ExternalPaymentMethodActivity : AppCompatActivity() {
```

Comment:
I wonder if we should rename this, maybe something like `ExternalPaymentMethodResultProxyActivity`, or something like that.

Reply:
I needed to fix some lint issues anyways, so I also did this. I renamed it to `ExternalPaymentMethodProxyActivity`, because it handles both the confirm + the result (so I didn't want to name it just `ResultProxyActivity`.